src/atoms/Panel/Panel.tsx

```diff
@@ -7,6 +7,7 @@ export const Panel = styled.section<{
   basic?: boolean;
   noPadding?: boolean;
   isPlaceholder?: boolean;
+  visible?: boolean;
```

Comment:
Do we need this? I'd rather keep prop types that are used in other components in those components.

Reply:
Ahh I added it there because it was giving me an error when I tried to add it originally because it wasn't defined in `Panel` which I compose with styled-components. 

I just tried though and it looks like adding this to `Drawer` worked
```export const Drawer = styled(Panel)<{ visible?: boolean }>```

I'll push this up and merge.